tests/features/sbom-explorer.feature

```diff
@@ -0,0 +1,178 @@
+    #       how we can 'loop' over all entries (seems should not be done in cucumber test description?)
+    When user selects any column, except description, to sort by
+    Then list of Vulnerabilities should be sorted by selected column
+
+  Scenario: View SBOM Vulnerability with long description
+    Given there is ingested SBOM which is affected by Vulnerability with long description
+
+    When user visits SBOM details page
+    And user selects Vulnerabilities tab
+    And user finds Vulnerability with long description
+    Then only shortened Vulnerability description should be visible
+    And 'Show more' link should be visible
+
+    When user selects the 'Show more' link
+    Then full expanded Vulnerability description should be visible
+    And 'Show less' link should be visible
+
+    When user selects the 'Show less' link
+    Then shortened Vulnerability description should be visible
+    And full Vulnerability description should not be visible
+
+  Scenario: View SBOM Vulnerability with relevant advisory
+    Given there is ingested SBOM which is affected by Vulnerability with relevant Advisory
+
+    When user visits SBOM details page
+    And user selects Vulnerabilities tab
+    And user finds Vulnerability with associated Advisory
+    And user selects the given Vulnerability (by clicking on CVE ID column)
+    Then link to All CVE details should be visible
+    and list of Relevant advisories should be visible
+    And link leading to details of the Advisory should be visible
+
+  Scenario: View SBOM Vulnerability without relevant advisory
+    Given there is ingested SBOM which is affected by Vulnerability without relevant Advisory
+
+    When user visits SBOM details page
+    And user selects Vulnerabilities tab
+    And user finds Vulnerability with associated Advisory
+    And user selects the given Vulnerability (by clicking on CVE ID column)
+    Then link to All CVE details should be visible
+    And list of Relevant advisories should not be visible
+
+  # TODO: clarify if this is also expected in V2
+  # also - if CVE is listed under SBOM Vulns. does it imply that to always have at least one affected dependency/package?
+  #
+  # Scenario: View SBOM Vulnerability with affected dependency
+  #   Given there is ingested SBOM which is affected by Vulnerability with affected dependency
+  #
+  #   When user visits SBOM details page
+  #   And user selects Vulnerabilities tab
+  #   And user finds Vulnerability with affected dependency
+  #   And user selects to see given Vulnerability Affected dependencis (by clicking on the count of Affected dependencies)
+  #   Then list of affected packages should be visible
+  #   And each entry should have expected fields (Type, Namespace, Name, Version, Path, Qualifiers)
+  #   And each entry name should be link to that Package details page
+  #   # And each entry should be expandable - to what - currently it says just 'Only direct dependencies'?
+
```

Comment:
This needs to be clarified

Reply:
Yes, as also all the other TODOs, but that as there was no time for that yet seems to me will have to be done later.